python/cudf/cudf/tests/test_groupby.py

```diff
@@ -1490,12 +1490,7 @@ def test_groupby_datetime_multi_agg_multi_groupby():
     ],
 )
 def test_groupby_multi_agg_hash_groupby(agg):
-    alphabets = "abcdefghijklmnopqrstuvwxyz"
-    prefixes = alphabets[:10]
-    coll_dict = dict()
-    for prefix in prefixes:
-        for this_name in alphabets:
-            coll_dict[prefix + this_name] = float
+    coll_dict = {letter: float for letter in string.ascii_lowercase}
```

Comment:
What is float here? Do we have some global fixture called float? If so, can we... not? If not, where is this defined?

Reply:
The `dtype` parameter of `cudf.datasets.timeseries` below takes a `dict[Hashable, type]` i.e. a mapping of column label to the column type when generating data.

Guessing this is one of the (many) legacy ways to generate test data

Reply:
Oh lol this is literally the builtin type float. I was off looking for a variable. Never mind, this is fine, carry on.